pkg/machines/vmnetworktab.jsx

```diff
@@ -59,11 +62,11 @@ class VmNetworkTab extends React.Component {
 
         // Network data mapping to rows
         const detailMap = [
-            { name: _("Type"), value: (network) => rephraseUI('networkType', network.type), header: true },
+            { name: _("Type"), value: (network, networkId) => <div id={`${id}-network-${networkId}-type`}>{rephraseUI('networkType', network.type)}</div>, header: true },
```

Comment:
just nitpicking, no need to change: for the sake of readability, the `div` can be instantiated outside this array composition.